docs/types/mutations.rst

```diff
@@ -38,20 +39,37 @@ So, we can finish our schema like this:
 
 .. code:: python
 
-    # ... the Mutation Class
+    from graphene import ObjectType, Mutation, String, Boolean, Field, Int, Schema
+
+
+    class Person(ObjectType):
+        name = String()
+        age = Int()
+
+
+    class CreatePerson(Mutation):
+        class Arguments:
+            name = String()
+
+        ok = Boolean()
+        person = Field(lambda: Person)
+
+        def mutate(self, info, name):
```

Comment:
```suggestion
        def mutate(root, info, name):
```

Reply:
The no-code done yet, but another good catch.

Reply:
I think the topic of `what is the name of the first argument of a resolver called` is its own docs related discussion/issue. Its definitely not `self`, so this is a good change IMO.

Reply:
Somewhere in the ecosystem, in an `Issue`, there's a great discussion regarding `root` as best, but for the life of me I can't find it again. If I do, I'll pop it into the docs tracking issue.